src/Microsoft.ML.Data/Evaluators/MultiOutputRegressionEvaluator.cs

```diff
@@ -556,7 +556,7 @@ private protected override Delegate[] CreateGettersCore(Row input, Func<int, boo
 
             t = schema[ScoreIndex].Type as VectorType;
             if (t == null || !t.IsKnownSize || t.ItemType != NumberType.Float)
-                throw Host.Except("Score column '{0}' has type '{1}' but must be a known length vector of type R4", ScoreCol, t);
+                throw Host.Except(nameof(schema), "score", ScoreCol, "known-size vector of float", t.ToString());
```

Comment:
>Except [](start = 27, length = 6)

Inconsistent: You are not using `ExceptSchemaMismatch` here as I see you used it in `QuantileRegressionEvaluator.cs` at line 453 for the same type of error.

Reply:
Thanks for the catch!

---
In reply to: [250753412](https://github.com/dotnet/machinelearning/pull/2217#discussion_r250753412) [](ancestors = 250753412)